content/docs/contributing/to-docs/style-guide/format.md

```diff
@@ -251,73 +167,73 @@ When the quote is a complete sentence, place the punctuation inside the quote.
-Open the `/<project>/game.cfg` file. | Open the /\<project\>/game.cfg file.
+### Tables
 
-### Code style for inline code and commands
+Table example:
 
-Do | Don't
-:--| :-----
-... set `enable_memory_tracking = True`. | ... set **enable_memory_tracking** = "True".
-... enter the command `dump_vars`. | ... enter the command dump_vars.
+```markdown
+| Default column| Right-aligned column | Center-aligned column | Left-aligned column |
+| - | -: | :-: | :- |
+| Row | entry | entry | entry |
+| Row | entry | entry | entry |
+| Row with missing entry | entry | | entry |
+```
 
-### Code style for variable names, API objects, namespaces, and so on
+Table result:
 
-Do | Don't
-:--| :-----
-... value of the `sys_maxfps` field. | ... value of the "sys_maxfps" field.
-... use the `WorldRequestBus`. | ... use the "WorldRequestBus".
-... in `AZ::Data::AssetData` derived classes. | ... in **AZ::Data::AssetData** derived classes.
+| Default column| Right-aligned column | Center-aligned column | Left-aligned column |
+| - | -: | :-: | :- |
+| Row | entry | entry | entry |
+| Row | entry | entry | entry |
+| Row with missing entry | entry | | entry |
 
-### Bold property names, and inline code their values
+### Tabs
 
-Do | Don't
-:--| :-----
-Set the **Color** property to `255,0,0`. | Set the Color property to "255,0,0".
-For **Intensity Mode**, select `Candela`. | For `Intensity Mode`, select Candela.
-Valid **Mass** values range from `0` to `Infinity`. | Valid Mass values range from `0` to Infinity.
+Tabs example:
 
-### Don't include the command prompt
 
-Do | Don't
-:--| :-----
-`cmake --build ...` | `C:\> cmake --build ...`
+```
+{{</* tabs name="name-for-this-group-of-tabs" */>}}
+{{%/* tab name="First tab" */%}}
 
-### Separate commands from output
+First tab's content.
 
-```shell
-cmake --build <MyProject> --target Editor --config profile -- -m
+{{%/* /tab */%}}
+{{%/* tab name="Second tab" */%}}
+
+Second tab's content.
+
+{{%/* /tab */%}}
+{{</* /tabs */>}}
 ```
 
-The output is similar to this:
 
-```console
-Microsoft (R) Build Engine version 16.9.0+57a23d249 for .NET Framework
-Copyright (C) Microsoft Corporation. All rights reserved.
+Tabs result:
 
-  Checking Build System
-  Building Custom Rule D:/O3DE/Code/Framework/AzCore/CMakeLists.txt
-  unity_16_cxx.cxx
-  unity_20_cxx.cxx
-  unity_19_cxx.cxx
-  unity_24_cxx.cxx
-  unity_23_cxx.cxx
-```
+{{< tabs name="tabs-example" >}}
+{{% tab name="First tab" %}}
 
-## Lists
+First tab's content.
+
+{{% /tab %}}
+{{% tab name="Second tab" %}}
+
+Second tab's content.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+### Lists
 
 Use ordered lists when the order of the items is significant, such as a procedure of sequential steps. Use unordered lists if the order of the items is arbitrary, such as a list of assets.
 
 If the list is longer than four items, or if the list contains call-out shortcode or an image, add newlines between each list element to improve readability.
```

Comment:
This looks like it should be moved into the `Ordered lists` section?

Reply:
Reorganizing this section!